Dockerfile

```diff
@@ -0,0 +1,46 @@
+# An integration test & dev container which builds and installs libgdf & pygdf from master
+
+FROM nvidia/cuda:9.2-devel
+# Or, change base image to for your installed CUDA driver version
+# Tested working with
+#FROM nvidia/cuda:9.1-devel
+#FROM nvidia/cuda:9.0-devel
+
+RUN apt update -y --fix-missing && \
+    apt upgrade -y && \
+    apt install -y \
+      git \
+      build-essential \
+      python3-dev \
+      gcc-4.8 \
+      g++-4.8
+
+# Install conda
+ADD https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh /miniconda.sh
+RUN sh /miniconda.sh -b -p /conda && /conda/bin/conda update -n base conda
+ENV PATH=${PATH}:/conda/bin
+
+# Enables "source activate conda"
+SHELL ["/bin/bash", "-c"]
+
+# Combined libgdf/pygdf conda env
+ADD conda_environments/gdf_build.yml /conda_environments/gdf_build.yml
+# Also tested working with 3.5
+ENV PYTHON_VERSION=3.6
+RUN conda env create -n gdf python=${PYTHON_VERSION} --file /conda_environments/gdf_build.yml
+
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64:/usr/local/lib
+ENV CC=/usr/bin/gcc-4.8
+ENV CXX=/usr/bin/g++-4.8
+
+RUN git clone --recurse-submodules https://github.com/gpuopenanalytics/libgdf
+RUN git clone --recurse-submodules https://github.com/gpuopenanalytics/pygdf
```

Comment:
This assumes you want the master branch of both libgdf and pygdf. I imagine that won't always be the case, but I can't come up with any good way to address this within a Dockerfile. @randerzander any thoughts?

Reply:
'm going to try Docker build-args and see if they'll do the trick: https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg